scenes/game/character.gd

```diff
@@ -74,19 +80,29 @@ func set_exceed(is_exceed : bool):
 func get_size():
 	return animation.sprite_frames.get_frame_texture("idle", 0).get_size()
 
+func set_buddy_id(id : String):
+	buddy_id = id
+
+func get_buddy_id():
+	return buddy_id
+
+func play_animation(named_animation : String):
+	if animation.sprite_frames.has_animation(named_animation):
+		animation.play(named_animation)
```

Comment:
not at all important, but it would be nice if something here could have have run/dash_back play the walking animations instead if they're not available